joomla/joomla-cms
# Add description to a move/copy operation in batch process form

### Summary of Changes

This PR will add description to a move/copy label text in batch process modal forms for Content, Menus, Banners, News feeds and Tags. ATM all controls except move/copy have a description on that form.
### Testing Instructions

Go to Content->Articles, select item and go to batch modal form, hover over label above move/copy category selection control - there wont be any tooltip.
Go to Menus -> All menus items, select an item, go to batch modal form, hover over label above move/copy category selection control - there wont be any tooltip.

Apply this PR - again check above instructions - new description tooltip should be visible.

Same instruction for Content->Categories, Banners, Banners->Categories, Contacts, Contacts->Categories, News feeds, News feeds->Categories.
### Documentation Changes Required

Modified key JLIB_HTML_BATCH_MENU_LABEL value in en-GB.lib_joomla.ini and en-GB.com_menus.ini

Added new keys JLIB_HTML_BATCH_MENU_LABEL_DESC to en-GB.lib_joomla.ini and en-GB.com_menus.ini

Modified/added key strings may need some adjustments to correct grammar/syntax.
